packages/next/src/server/config.test.ts

```diff
@@ -146,6 +146,20 @@ describe('loadConfig', () => {
         /The experimental feature "experimental.turbopackPersistentCaching" can only be enabled when using the latest canary version of Next.js./
       )
     })
+
+    it('errors when using persistentCachingForBuild if not in canary', async () => {
```

Comment:
i think we also want to remove the canary block

@feedthejim

Reply:
We can do that in a separate PR too